commit fc1454485c9e2ea6e3a5bfef5a941787772efd79
Author: Ralf Haferkamp <rhafer@suse.de>
Date:   Wed May 7 14:53:22 2014 +0200

    HACKS or Demo setup
    
    - most of the repos are unsigned there
    - the DEPS image doesn't provide any patterns

diff --git a/chef/cookbooks/provisioner/templates/suse/crowbar_register.erb b/chef/cookbooks/provisioner/templates/suse/crowbar_register.erb
index d59c3d1..fe1cc76 100644
--- a/chef/cookbooks/provisioner/templates/suse/crowbar_register.erb
+++ b/chef/cookbooks/provisioner/templates/suse/crowbar_register.erb
@@ -156,7 +156,7 @@ zypper ar "<%= @repos[name][:url] %>" "<%= name %>"
 zypper --gpg-auto-import-keys refresh -r SLE-Cloud-PTF
 
 ZYPPER_REF_OPT=
-test $CROWBAR_AUTO_IMPORT_KEYS -eq 1 && ZYPPER_REF_OPT=--gpg-auto-import-keys
+test $CROWBAR_AUTO_IMPORT_KEYS -eq 1 && ZYPPER_REF_OPT="--gpg-auto-import-keys --no-gpg-checks"
 zypper $ZYPPER_REF_OPT refresh
 
 
@@ -176,7 +176,7 @@ PACKAGES_INSTALL="$PACKAGES_INSTALL biosdevname netcat rubygem-cstruct supportut
 # From sleshammer
 PACKAGES_INSTALL="$PACKAGES_INSTALL rubygem-chef rubygem-rest-client rubygem-cstruct rubygem-libxml-ruby rubygem-wsman rubygem-xml-simple parted lvm2 ipmitool curl tack nfs-client rsyslog ntp tcpdump"
 
-zypper --non-interactive install -t pattern $PATTERNS_INSTALL
+# zypper --non-interactive install -t pattern $PATTERNS_INSTALL
 # Auto-agree with the license since it was already agreed on for the admin server
 zypper --non-interactive install --auto-agree-with-licenses suse-cloud-release
 zypper --non-interactive install $PACKAGES_INSTALL
